apps/participants/views.py

```diff
@@ -100,7 +100,7 @@ def get_context_data(self, *args, **kwargs):
             sessions = participant.experimentsession_set.filter(experiment=experiment).all()
             experiment_data[experiment] = {
                 "sessions": sessions,
-                "participant_data": json.dumps(sessions.first().participant_data_from_experiment),
+                "participant_data": json.dumps(sessions.first().participant_data_from_experiment, indent=4),
```

Comment:
![image](https://github.com/dimagi/open-chat-studio/assets/64970009/bbc5b57b-2187-43ae-bde1-ff5d4a271a87)

Reply:
How about using the same view as in the session details page: https://github.com/dimagi/open-chat-studio/blob/bc07355c9d567020e4cfdee52c21d80be89e477a/templates/experiments/experiment_session_view.html#L45-L71

Reply:
We _can_, but that view isn't editable. Just making sure we're aligned that it should be editable. We can make the json view editable and the non-json view non-editable and let it serve as a preview?

Reply:
right, fine to leave it as is for now.